pkg/capi/edit/cluster.x-k8s.io.cluster/ClusterConfig.vue

```diff
@@ -61,7 +65,13 @@ export default {
     }
   },
   beforeMount() {
-    this.initSpecs();
+    this.initSpecs().then((namespaces) => {
+      this.allNamespaces = namespaces || [];
```

Comment:
Why initialize some things in `initSpecs` but return namespaces and set them in `beforeMount`?